client/src/app/local-api/indicators/route.ts

```diff
@@ -5,7 +5,7 @@ import { IndicatorView } from "@/app/parsers";
 
 import INDICATORS from "./indicators.json";
 
-export type VisualizationTypes = "map" | "table" | "chart" | "numeric" | "ai";
+export type VisualizationTypes = "map" | "table" | "chart" | "numeric" | "ai" | "custom";
```

Comment:
We don't need this, otherwise you need to create query_custom and all the props drilling and all the changes across the app. We have already a way to handle custom components, check total-area